src/statistics/data.jl

```diff
@@ -60,9 +60,11 @@ EmpiricalHistogram(d, s::Number) = EmpiricalHistogram(d, BlockWeighting(ntuple(i
     EmpiricalHistogram(sdata)
     EmpiricalHistogram(sdata, v)
     EmpiricalHistogram(sdata, v, s)
+    EmpiricalHistogram(sdata, v, s; kwargs...)
```

Comment:
Let's try to add the kwargs to all methods, it should be very similar to what you already did for the first two methods 👍🏽